node/src/components/linear_chain_sync/error.rs

```diff
@@ -76,7 +76,7 @@ pub(crate) enum LinearChainSyncError {
     },
 
     #[error(
-        "Current version is {current_version}, but current block header has older version: \
+        "Network is still running an older version. Current version is {current_version}, but current block header has older version: \
```

Comment:
Line too long

Reply:
Addressed in https://github.com/casper-network/casper-node/pull/2434/commits/f89f8e38fe57b187ece6b627a8e76045c93e61e2